README.md

```diff
@@ -23,7 +23,7 @@ Please see the [FAQ][] for answers about AKS Engine and its progenitor ACS-Engin
 
 ## Join the community
 
-Want to get involved? The [community guide][community] covers everything you need to know about the AKS Engine community and how you can contribute. The [developer guide][developer-guide] will help you onboard as a developer.
+If you are committed to using AKS Engine longer term and would like to become a project maintainer, please reach out to us via the [#aks-engine-dev Slack channel](https://kubernetes.slack.com/archives/CU1CXUHN0)! The [community guide][community] covers everything you need to know about the AKS Engine community and how you can contribute. The [developer guide][developer-guide] will help you onboard as a developer. The AKS Engine community is committed to integrating and validating new versions of Kubernetes into AKS Engine. We encourage AKS Engine users to evaluate moving to CAPZ as it provides stronger support for managing the cluster lifecycle compared to AKS Engine.
```

Comment:
Is CAPZ ready for production work?  (At least at the level of aks-engine?)  I would be careful how we phrase the push to CAPZ.  I really want people to look at and play with CAPZ but are we ready for major customers to make production clusters that way?  (Just asking - I would love it if that were the case)

Reply:
CAPZ is ready for production work. Does it (or will it ever have) all the features of aks-engine? No, it likely will not